src/containers/issue.py

```diff
@@ -0,0 +1,100 @@
+import re
+from typing import List, Optional
+
+from .repository import Repository
+from .milestone import Milestone
+
+
+class Issue:
+    def __init__(self, repository: Repository):
+        self.number: int = 0
+        self.organization_name: str = repository.organization_name
+        self.repository_name: str = repository.repository_name
+        self.title: str = ""
+        self.state: str = ""
+        self.url: str = ""
+        self.body: str = ""
+        self.created_at: str = ""
+        self.updated_at: str = ""
+        self.closed_at: Optional[str] = None
+        self.milestone_number: Optional[int] = None
```

Comment:
Why the Milestone class is not used here?

Reply:
Milestones are removed from current project. Comment ready to be resolved.